Revisit create use case guide (#95)
superfaceai/docs: docs/guides/run-use-case.md

# Run use case

> This guide describes how a use case can be used in any production Node.js application.

Reviewer: ```suggestion
This guide describes how to start using a locally developed use case in a production Node.js application.
```

:::

## Compile Comlink source files to AST

Comlink profile and map files (.supr and .suma) needs to be compiled to their AST form (with .ast.json extension).

Run the following command to perform a one-off compilation:

```shell

> superface compile
> # or, if you don't have the CLI installed
> npx @superfaceai/cli@latest compile

Reviewer: I think this is too many options, let's just go with npx here?

```suggestion
npx @superfaceai/cli@latest compile
```

```

This will generate `.ast.json` files next to the existing source files linked from `super.json` file.

:::caution

> The `.ast.json` files must be available in the runtime for OneSDK. It is recommended to commit the `.ast.json` files to version control.

Reviewer: ```suggestion
The `.ast.json` files must be available in the runtime for OneSDK. We recommend to commit the `.ast.json` files to version control.
```

:::

> The compilation is **necessary after every** change to local `.suma` and `.supr` files.

Reviewer: Highlight also the noun.

```suggestion
The compilation is **necessary after every change** to local `.suma` and `.supr` files.
```

Consider also moving this to the callout above or its own.
